### PR TITLE
add modern_botok with a new dictionary.

### PR DESCRIPTION
**Pull request by ykyogoku**
The dictionary is compiled from various dictionaries taken from Christian Steinert's repo and adds support for modern Tibetan language

### CODE REVIEW

**Review by eroux — CHANGES_REQUESTED** [review]
thanks @ykyogoku ! A few things:
- can you add a README.md file in the dialect_packs/modern_botok/ directory with some background on the files? (acknowledging the project, contributors, etc.)
- can you update the main README and add references to this new dialeck_pack, with a link to the more detailed README?

File: dialect_packs/modern_botok/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# [modern-botok](https://github.com/Divergent-Discourses/modern-botok)
+This repository offers a customised dictionary (dictionary/tsikchen.tsv) and contains an explanation of how to integrate the customised dictionary into BoTok.
+Botok out-of-the-box can tokenize classical Tibetan text or traditional genres, however since it depends on a dictionary for tokenization, lacks capabilities for modern Tibetan, in particular, the language of modern newspapers published in the PRC or on the sub-continent. Adding this customised dictionary adds functionality for modern Tibetan to BoTok.
```
Comment (eroux) [inline dialect_packs/modern_botok/README.md]: this seems a bit out of place, please remove

File: dialect_packs/modern_botok/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  2. [Jim Valby](https://github.com/christiansteinert/tibetan-dictionary/blob/master/_input/dictionaries/public/07-JimValby)
+  2. [Ives Waldo](https://github.com/christiansteinert/tibetan-dictionary/blob/master/_input/dictionaries/public/08-IvesWaldo)
+  3. [Dan Martin](https://github.com/christiansteinert/tibetan-dictionary/blob/master/_input/dictionaries/public/09-DanMartin)
+  4. [Tshig mdzod chen mo](https://github.com/christiansteinert/tibetan-dictionary/blob/master/_input/dictionaries/public/25-tshig-mdzod-chen-mo-Tib)
+  5. [Dung dkar](https://github.com/christiansteinert/tibetan-dictionary/blob/master/_input/dictionaries/public/34-dung-dkar-tshig-mdzod-chen-mo-Tib)
+  6. [Tibetan Terminology Project](https://github.com/christiansteinert/tibetan-dictionary/blob/master/_input/dictionaries/public/48-TibTermProject
+)
+- The resulting dictionary was cleaned up and edited by the Divergent Discourses project to the project's requirements (removal of double entries, phraseologisms, ungrammatical entries, etc; addition of ca. 1000 personal and place names)
+
+## Copyright
```
Comment (eroux) [inline dialect_packs/modern_botok/README.md]: Please replace with "acknowledgement"

File: dialect_packs/modern_botok/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+The **modern_botok dictionary** was complied by Franz Xaver Erhard, Yuki Kyogoku and Xiaoying of Leipzig University for the Divergent Discourses project. The project is a joint study involving SOAS University of London and Leipzig University, funded by the AHRC in the UK and the DFG in Germany.
+
+Please acknowledge the project in any use of these materials. Copyright for the project resides with the two universities.
```
Comment (eroux) [inline dialect_packs/modern_botok/README.md]: please remove "copyright for the project...", I'm not sure what that even means... I doubt SOAS owns the copyright on the Tsik mdzod chen mo

File: dialect_packs/modern_botok/README.md [hunk 4]
```diff
@@ -15,6 +15,6 @@ Botok out-of-the-box can tokenize classical Tibetan text or traditional genres,
 - The resulting dictionary was cleaned up and edited by the Divergent Discourses project to the project's requirements (removal of double entries, phraseologisms, ungrammatical entries, etc; addition of ca. 1000 personal and place names)
 
 ## Copyright
-The **modern_botok dictionary** was complied by Franz Xaver Erhard, Yuki Kyogoku and Xiaoying of Leipzig University for the Divergent Discourses project. The project is a joint study involving SOAS University of London and Leipzig University, funded by the AHRC in the UK and the DFG in Germany.
+The **modern_botok dictionary** was complied by [Franz Xaver Erhard](https://github.com/ykyogoku), [Yuki Kyogoku](https://github.com/fxerhard), and 笑影 Xiao Ying of Leipzig University for the [Divergent Discourses](https://github.com/Divergent-Discourses/) project. The project is a joint study involving SOAS University of London and Leipzig University, funded by the AHRC in the UK and the DFG in Germany.
 
 Please acknowledge the project in any use of these materials. Copyright for the project resides with the two universities.
```
Comment (eroux) [inline dialect_packs/modern_botok/README.md]: @fxerhard can you also remove this?

**Review by eroux — APPROVED** [review]

File: dialect_packs/modern_botok/README.md (hunk 1 above)
Comment (fxerhard) [reply dialect_packs/modern_botok/README.md]: removed the two paragraphs

File: dialect_packs/modern_botok/README.md (hunk 4 above)
Comment (fxerhard) [reply dialect_packs/modern_botok/README.md]: done so

File: dialect_packs/modern_botok/README.md (hunk 3 above)
Comment (fxerhard) [reply dialect_packs/modern_botok/README.md]: copyright removed

File: dialect_packs/modern_botok/README.md (hunk 2 above)
Comment (fxerhard) [reply dialect_packs/modern_botok/README.md]: done

**Pull request merged**